### DISCUSSION TITLE
Why ai this happening

### DISCUSSION THREAD

**Discussion by chuckbartok** [community/community#86752] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

One of our readers sent me this?
The link says website is not working>
![subs](https://github.com/community/community/assets/155028263/0769c6d4-8fc5-426e-b074-56c8944b8481)The link takes to this page
![subs1](https://github.com/community/community/assets/155028263/2b75f898-e8b6-4428-8c2f-678e75f3bc10)

**Comment by a-jean-andreasian** [comment]
The link is broken or the source was removed/deleted.

**Comment by a-jean-andreasian** [comment]
However, this email, and the link of it, look very suspicious to me.
